File: 프로그래머스/unrated/120868. 삼각형의 완성조건 （2）/삼각형의 완성조건 （2）.py
def solution(sides):
    answer = 0
    for i in range(1,max(sides)):
        if i + min(sides) > max(sides):
            answer += 1
            print(i)
    print()
    i = max(sides)
    while True:
        if sum(sides) > i:
            answer += 1
            i += 1
        else:
            break
        print(i)
    return answer

File: 프로그래머스/0/120868. 삼각형의 완성조건 （2）/삼각형의 완성조건 （2）.java
class Solution {
    public int solution(int[] sides) {
        int answer = 0;
        int max = Math.max(sides[0], sides[1]), min = Math.min(sides[0], sides[1]);
        
        
        //max가 가장 클 때 max < i + min, i < max
        int i = max;
        while(max < i + min){
            answer++;
            i--;
        }
        
        //i가 가장 클 때 i > max, i < min + max
        i = max + 1;
        while(i < min + max){
            answer++;
            i++;
        }
        
        
        return answer;
    }
}